src/components/playback/playbackmanager.js

```diff
@@ -1463,6 +1487,24 @@ class PlaybackManager {
             return getPlayerData(player).subtitleStreamIndex;
         };
 
+        self.getSecondarySubtitleStreamIndex = function (player) {
+            player = player || self._currentPlayer;
+
+            try {
+                if (player && !enableLocalPlaylistManagement(player)) {
+                    return player.getSecondarySubtitleStreamIndex();
+                }
+            } catch (e) {
+                console.error(`Failed to get secondary stream index: ${e}`);
```

Comment:
Very minor, but for debugging I like to include the filename in logs and the actual error should just be passed as a second param.

```suggestion
                console.error('[playbackmanager] Failed to get secondary stream index:', e);
```

Reply:
Yup sounds good!